execution/src/windows/execution.cc

```diff
@@ -380,7 +380,7 @@ namespace leatherman { namespace execution {
             if (options[execution_options::throw_on_nonzero_exit]) {
                 throw child_exit_exception("child process returned non-zero exit status.", 127, {}, {});
             }
-            return make_tuple(false, "", "");
+            return make_tuple(false, "", "", 127);
```

Comment:
Is 127 idiomatic for command not found in Windows land? https://msdn.microsoft.com/en-us/library/windows/desktop/ms681382(v=vs.85).aspx sez "The specified procedure could not be found." which may be the same thing - not sure.

Mostly asking out of curiosity, because being consistent across windows/posix is obviously nice to have.